assets/mixs-patterns-materialized.yaml

```diff
@@ -23983,7 +23983,7 @@ classes:
       host_symbiont:
         name: host_symbiont
         examples:
-        - value: Paragordius varius
+        - value: Paragordius various
```

Comment:
this is an incorrect change, the value should remain as "varius" as its a species name.